test/support/slack_client.ex

```diff
@@ -73,9 +73,39 @@ defmodule Cog.Test.Support.SlackClient do
       nil ->
         raise RuntimeError, message: "$SLACK_USER_API_TOKEN not set!"
       token ->
-        {:ok, client} = start_link(token)
-        :ok = initialize(client)
-        {:ok, client}
+        api_wait()
+        api_wait()
```

Comment:
It does seem weird that this is called twice in a row. A note or a `:timer.sleep(@api_wait_interval * 2)` might be more intention revealing.